### PR TITLE
#1 Create the pom and the helloWorld.kt

### PR DESCRIPTION
**Pull request by chris0197**
@nicolaiparlog  #16

### CODE REVIEW

**Comment by nipafx** [comment]
Create and link to an issue as described [here](https://github.com/0riginz/C0ffe1sWh8tWeNeed/pull/5#issuecomment-320639589).

File: Main.kt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package FinalProject
```
Comment (nipafx) [inline Main.kt:1]: Package names are by convention entirely lower cased (e.g. `finalproject` or maybe `final_project`). Generally speaking, something like `traffic` might make more sense.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-stdlib</artifactId>
+            <version>${kotlin.version}</version>
+        </dependency>
+
+
+    </dependencies>
+
+    <build>
+        <sourceDirectory>${project.basedir}/src/main/kotlin</sourceDirectory>
```
Comment (nipafx) [inline pom.xml:30]: Here you define (correctly) where to find source files but `Main.kt` is not in there. Once you fix that, the build works.

**Review by nipafx — CHANGES_REQUESTED** [review]

**Comment by nipafx** [comment]
The README changes you added in 1280ffd do not belong into this pull request.

**Comment by chris0197** [comment]
@nicolaiparlog added some changes to the pom.xml

**Comment by nipafx** [comment]
I don't see it in this pull request.

**Comment by nipafx** [comment]
Also, you still need to link to the issue...

**Comment by chris0197** [comment]
@nicolaiparlog
 i changed the file over git and use mvn clean install (showed Hello World) after that i pushed the pom and src folder

**Comment by nipafx** [comment]
> after that i pushed the pom and src folder

But apparently not to this branch or you got an error message that you ignored, or... many things can go wrong but the fact is, the commit is not in this pull request.

Have a look at what's above [this comment](https://github.com/0riginz/C0ffe1sWh8tWeNeed/pull/6#issuecomment-320901422) (scroll up a little). What you see there is a commit that was added to this PR by pushing the `chris0197-patch-3` branch. Since then no new commits were added, so there are no changes for me to review.

**Comment by chris0197** [comment]
@nicolaiparlog forgot to commit...

**Comment by chris0197** [comment]
@nicolaiparlog 
commit and push were successfully merged by accident but should be right....
sorry

**Comment by nipafx** [comment]
You need to improve your Git/GitHub abilities. One rule of thumb: When you're trying to merge anything on the command line, chances are high, you're doing something wrong.

**Pull request closed without merging**